presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/MergeIdenticalWindows.java

```diff
@@ -26,11 +26,9 @@
 
```

Comment:
No, this commit adds the rule to the optimizer to collapse windows with identical `(partition, order)` but different frames (in opposite to `MergeIdenticalWindows` before which were collapsing only those windows with identical `(partition, order, frame)`). Previous commit (https://github.com/prestodb/presto/pull/6009/commits/5dd90a380a5ef0370d5b61896bb5f29283fe5446) enables executing multiple frames in EXECUTION but without this rewrite it's not useable. We have worked on that feature with @ebd2 at the same time to expedite delivery. That's why you see these commits distinct - Eric took care of optimizer and I took care of execution part.